src/ghga_transpiler/cli.py

```diff
@@ -20,13 +20,20 @@
 
 import typer
 
-from . import io
+from . import __version__, io
 from .config.exceptions import UnknownVersionError
 from .core import convert_workbook
 
 cli = typer.Typer()
 
 
+def version_callback(value: bool):
+    """Prints the package version"""
+    if value:
+        print(f"ghga-transpiler version: {__version__}")
```

Comment:
```suggestion
        print(__version__)
```
IMHO it's good practice to make these output machine readable, since the text around doesn't offer much additional value